packages/gemacs.rb

```diff
@@ -3,53 +3,90 @@
 class Gemacs < Package
   description 'An extensible, customizable, free/libre text editor - and more.'
   homepage 'https://www.gnu.org/software/emacs/'
-  @_ver = '27.1'
-  version "#{@_ver}-2"
+  @_ver = '28.0.50.1'
+  version "#{@_ver}-7a7b"
   compatibility 'all'
-  source_url "https://ftpmirror.gnu.org/emacs/emacs-#{@_ver}.tar.xz"
-  source_sha256 '4a4c128f915fc937d61edfc273c98106711b540c9be3cd5d2e2b9b5b2f172e41'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-27.1-2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-27.1-2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-27.1-2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-27.1-2-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-28.0.50.1-7a7b-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-28.0.50.1-7a7b-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-28.0.50.1-7a7b-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gemacs-28.0.50.1-7a7b-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: '270c65753da8ff73172aaecf8696eef4645204904d2345620ea54d24a880a814',
-     armv7l: '270c65753da8ff73172aaecf8696eef4645204904d2345620ea54d24a880a814',
-       i686: '2b94b8c5016dbf1e39cdaf4d78e8678a9414232bac93701e0ba2e10e7864b428',
-     x86_64: '5055031715b416892f0da3c5d3faf0ff535e1ca74d57be96e8d2c0956de79794'
+    aarch64: '74066f8609ca1403609962454feff1a85aedb24a3d56dbfcae4a3aad8d00f836',
+     armv7l: '74066f8609ca1403609962454feff1a85aedb24a3d56dbfcae4a3aad8d00f836',
+       i686: '401243600795df8a6c375a7e581b939e258d1697595e9a43573860fca34656c0',
+     x86_64: '702ac1d56095b1032d387a2e4736e6f8ac49d4d16a165cbc1142cb5ffe394454'
   })
 
+  depends_on 'alsa_lib'
+  depends_on 'cairo'
   depends_on 'giflib'
+  depends_on 'gpm'
+  depends_on 'gtk3'
+  depends_on 'harfbuzz'
+  depends_on 'jansson'
   depends_on 'libjpeg'
+  depends_on 'libotf'
   depends_on 'libpng'
+  depends_on 'librsvg'
   depends_on 'libtiff'
-  depends_on 'imagemagick7'
-  depends_on 'sommelier'
+  depends_on 'texinfo'
+  depends_on 'webkit2gtk'
+
+  def self.prebuild
+    @git_dir = 'emacs_git'
+    @git_branch = 'feature/pgtk'
+    @git_hash = '7a7bc15242896b20c7af49f77f0e22c3d78e4d88'
```

Comment:
Can we check out with a git tag instead?  Might be able to use `#{@_ver}` with the tag.

Reply:
@git_hash = '7a7bc15242896b20c7af49f77f0e22c3d78e4d88' is the git tag as I understand it.

Reply:
I don't know where you are getting this version but from what I can tell, the latest tagged release is:
```
fda9d916712db6a1262cdb4539eeb4665b6cefb7	refs/tags/emacs-27.1.91
```
See https://git.savannah.gnu.org/git/emacs.git/info/refs.

Reply:
This is from the feature/pgtk branch (which enables full gtk support allowing usage from Wayland.)

https://git.savannah.gnu.org/cgit/emacs.git/log/?h=feature/pgtk